doc/time/zone_and_in.rdoc

```diff
@@ -1,8 +1,5 @@
-- +zone+: a timezone, which may be:
-  - A string offset from UTC.
-  - A single letter offset from UTC, in the range <tt>'A'..'Z'</tt>,
-    <tt>'J'</tt> (the so-called military timezone) excluded.
-  - An integer number of seconds.
-  - A timezone object;
-    see {Timezone Argument}[#class-Time-label-Timezone+Argument] for details.
-- <tt>in: zone</tt>: a timezone _zone_, which may be as above.
+- +zone+: a timezone +zone+.
```

Comment:
Same with this, it seems like there's only one place (in `Time.new` in `timev.rb`) that uses this file. I think we can just remove this file and directly embed the contents.

Reply:
Same.